src/nn/models.py

```diff
@@ -361,4 +361,13 @@ def set_covariate_weights(self, weights: numpy.ndarray):
         #('covariate weight shapes are ', weight.shape, snp_count, cov_weights.shape, weight[:, snp_count:].shape)
         weight[:, snp_count:] = cov_weights
         self.layer.weight = torch.nn.Parameter(weight)
-    
\ No newline at end of file
+
+class LassoNetClassifier(LassoNetRegressor):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        
+    def calculate_loss(self, y_hat: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
```

Comment:
maybe make a parameter for the loss with the default of binary_cross_entropy_with_logits? in case we have smth discrete but non-binary.

Reply:
and subtract 1 only if loss ==  binary_cross_entropy_with_logits
and if it's smth else, raise NotImplementedError ?